src/adapters/mc/CMakeLists.txt

```diff
@@ -3,5 +3,6 @@
 
 add_compile_options("-Wall;-Wextra;-Wunused;-Werror;-Wformat;-Wformat-security;-Wno-unused-result")
 
-add_subdirectory(ssh)
-add_subdirectory(asb)
\ No newline at end of file
+add_subdirectory(ssh)
+add_subdirectory(asb)
+add_subdirectory(complianceengine)
```

Comment:
Let's exclude it as per the offline discussion: https://github.com/Azure/azure-osconfig/commit/ce324cb40f462afaec036d29830ab7d105893db7

Reply:
My understanding is that we will exclude it from the build. But fine to do it in a separate PR. @jepio?

Reply:
Compliance Engine is *not* included in either the ASB of SSH builds, not in compiled binaries, not in packages. The code is just validated to be built at the same time but is not included anywhere. We do not want to include additional manual changes into this big merge PR, what's in this branch must be identical with dev at the time of merge. As explained, if so it will be decided, we will create a separate small PR to explicitly disable Compliance Engine from being compiled, that will likely come with a cost, but in that way, we can later easily revert that small PR to re-add it to build.

Reply:
This commit shows all the required changes to keep testing/fuzzing working: https://github.com/Azure/azure-osconfig/pull/1046/commits/b617016d34a780b72123b5266299a5dab484cbae

Reply:
@jepio tried those identically and they were not enough to keep CI working. Will redo in a clean next PR and if the issues will show there, we will use that to diagnose. We will keep this PR purely for the merge.

@jepio, are you OK with merging this once we tag main for GA?

Reply:
GA for ASB audit is now tagged in main: 

Tag: asb_audit_ga 
At commit: 0ab71b941de98b2ef7d277a574ae3f8087f561fb

We are ready to merge this and next PR which disables Compliance Engine from build as soon as we get alignment.

@jepio please comment.